platform/platform-configuration-listener/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -22,6 +22,7 @@
             <ext:property name="configFileDirectory" value="${ddf.home}/etc"/>
             <ext:property name="processedDirectory" value="${ddf.home}/etc/processed"/>
             <ext:property name="failedDirectory" value="${ddf.home}/etc/failed"/>
+            <ext:property name="exportedDirectory" value="${ddf.home}/etc/exported"/>
```

Comment:
Why is the output directory configured in this manner? It seems somewhat roundabout from a user perspective. Wouldn't a reasonable default with the ability to specify an output directory as a command line parameter make more sense? It's what I would  expect based on how other tools and commands work.